demo/x11_rawfb/nuklear_rawfb.h

```diff
@@ -765,6 +846,10 @@ nk_rawfb_stretch_image(const struct rawfb_image *dst,
                     continue;
             }
             col = nk_image_getpixel(src, (int)xoff, (int) yoff);
+	    /* This assumes the font atlas uses transparent black for non-glyph pixels */
```

Comment:
Could you please also add this comment to the font baking code to make this assumption visible?

Reply:
I ended up simply removing this comment because ultimately, it doesn't matter. Either the pixel is 100% transparent, in which case it doesn't matter what color it is, or it's not, and it should be given the foreground color specified, leaving the alpha channel as given by the atlas.